metrics/web/test/project_groups/presentation/widgets/project_group_card_test.dart

```diff
@@ -141,46 +268,360 @@ void main() {
     testWidgets(
       "shows the delete button if the widget is hovered",
       (WidgetTester tester) async {
-        await mockNetworkImagesFor(() {
-          return tester.pumpWidget(const _ProjectGroupCardTestbed(
-            theme: testTheme,
+        await tester.pumpWidget(const _ProjectGroupCardTestbed(
+          theme: testTheme,
+          projectGroupCardViewModel: projectGroupCardViewModel,
+        ));
+
+        await mockNetworkImagesFor(() async {
+          await _hoverProjectGroupCard(tester);
+        });
+
+        expect(find.text(CommonStrings.delete), findsOneWidget);
+      },
+    );
+
+    testWidgets(
+      "displays a name of the project group card view model",
```

Comment:
```suggestion
      "displays a name",
```
maybe simplify it a bit to improve readability?